lib/util.js

```diff
@@ -119,12 +119,14 @@ const util = {
       parseMatch();
     }
 
-    return ({
+    const result = {
       command: command,
       match: match,
       matchArgs: matchArgs,
       pipes: pipes
-    });
```

Comment:
I think we can introduce the parser here. `bash-parser` could substitute part of the actual function code to provide same results.